mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go

```diff
@@ -304,6 +304,11 @@ func TestAccBigQueryBigLakeManagedTable(t *testing.T) {
 			{
 				Config: testAccBigLakeManagedTable(bucketName, connectionID, datasetID, tableID, TEST_SIMPLE_CSV_SCHEMA),
 			},
+			{
+				ResourceName:      "google_bigquery_table.test",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
Looks like we need an ignore here on deletion_protection

Reply:
done.